### DISCUSSION TITLE
Cant expand qTree on label click if selected.sync active

### DISCUSSION THREAD

**Discussion by Code-Case** [quasarframework/quasar#9341] [General - Components / Directives / etc] [answered]
Hello :)

I like to get the selected node to do some css on them like
`    .q-tree__node--selected {
      color: red;
    }
`

but then I can expand the node only if I click the fancy but small arrow Icon, how can I change this, that the default behavior is that I can expand it, also if I click on the header/label?

Here is a codepen with 2 trees, left one is selcted.synced and right one isnt, but then without its selected css class.

https://codepen.io/someOne11/pen/dyvOJbO?editors=1111

**Comment by metalsadman** [comment, marked as answer]
https://codepen.io/metalsadman/pen/poeNape?editors=1010

**Reply by Code-Case** [reply]
Thats very nice, thanks!
